From 1d19134fdf24a6e6c2bd41b045d337d2ad763dd2 Mon Sep 17 00:00:00 2001
From: Alexander Couzens <lynxis@fe80.eu>
Date: Mon, 26 Jan 2026 22:49:32 +0100
Subject: [PATCH] Fix: conftest with pytest 9.0

The hook pytest_ignore_collect() has changed the arguments, path has been deprecated in 7.0 and dropped
in 9.0.
See https://docs.pytest.org/en/latest/deprecations.html#py-path-local-arguments-for-hooks-replaced-with-pathlib-path
for further information.
---
 conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conftest.py b/conftest.py
index 4c2c0c631a5c..cbb1e03f4a67 100644
--- a/conftest.py
+++ b/conftest.py
@@ -26,7 +26,7 @@ def add_doctest_context(doctest_namespace):  # noqa: PT004
     doctest_namespace["asyncio"] = ContribAsyncio()
 
 
-def pytest_ignore_collect(collection_path, path, config):
+def pytest_ignore_collect(collection_path, config):
     if sys.version_info >= (3, 10):  # noqa: UP036
         return None
 
-- 
2.52.0

